tiledb/tests/test_vfs.py

```diff
@@ -292,7 +292,8 @@ def test_ls(self):
         )
 
     @pytest.mark.skipif(
-        pytest.tiledb_vfs not in ["s3", "file"], reason="Only test on S3 and local"
+        pytest.tiledb_vfs not in ["file", "s3", "azure"],
```

Comment:
I don't think we have a harness set up to be able to test against these other ones?